tests/jwksClient.tests.js

```diff
@@ -182,8 +183,51 @@ describe("JwksClient", () => {
         done();
       });
     });
-  });
 
```

Comment:
You'll want to test to make sure the proxy is actually reached.  With the test you wrote here, if we remove the proxy option from the jwksClient, your test still passes.  Below I've written a test that will cover the proxy is actually reached

```suggestion

    it('should add a proxy', done => {
      const expectedError = { message: 'expectedError' };
    
      nock(`http://${proxy}`)
        .get(() => true)
        .replyWithError(expectedError);

      const client = new JwksClient({
        jwksUri: `${jwksHost}/.well-known/jwks.json`,
        requestHeaders: {
          'User-Agent': 'My-bot'
        },
        proxy: `http://username:password@${proxy}`
      });
  
      client.getKeys((err) => {
        expect(err).to.equal(expectedError);
        done();
      });
    });
```